scipy/interpolate/_bsplines.py

```diff
@@ -1441,20 +1442,22 @@ def make_lsq_spline(x, y, t, k=3, w=None, axis=0, check_finite=True):
 
     Returns
     -------
-    b : a BSpline object of the degree `k` with knots `t`.
+    b : a BSpline object of the degree ``k`` with knots ``t``.
 
     Notes
     -----
-    The number of data points must be larger than the spline degree `k`.
+    The number of data points must be larger than the spline degree ``k``.
 
-    Knots `t` must satisfy the Schoenberg-Whitney conditions,
+    Knots ``t`` must satisfy the Schoenberg-Whitney conditions,
     i.e., there must be a subset of data points ``x[j]`` such that
     ``t[j] < x[j] < t[j+k+1]``, for ``j=0, 1,...,n-k-2``.
 
     Examples
     --------
     Generate some noisy data:
+
```

Comment:
Because of no empty line, rendering is broken.

Before:
![SS 2022-08-20 at 9 23 43](https://user-images.githubusercontent.com/3813847/185722925-b531097d-33bd-47de-9984-c7e629905cd4.png)

After:
![SS 2022-08-20 at 9 41 29](https://user-images.githubusercontent.com/3813847/185722928-31c9b189-086d-4fb5-9d0e-5863e714b895.png)